### PR TITLE
Rename the chatters_refresh module

### PR DESCRIPTION
**Pull request by alazymeme**
- [x] `CHANGELOG.md` was updated, if applicable

### CODE REVIEW

**Comment by alazymeme** [comment]
I really need to squash commits when I merge master into my fork

File: pajbot/modules/__init__.py [hunk 1]
```diff
@@ -103,6 +102,7 @@
     LineFarmingModule,
     LinkCheckerModule,
     LinkTrackerModule,
+    ChattersRefreshModule,
```
Comment (pajlada) [inline pajbot/modules/__init__.py]: remove this resort - it is wrong. this list is sorted based on the name of the class, not any of its internal members

File: pajbot/modules/chatters_refresh.py [hunk 2]
```diff
@@ -14,10 +14,10 @@
 
 class ChattersRefreshModule(BaseModule):
     ID = __name__.split(".")[-1]
-    NAME = "Chatters refresh"
-    DESCRIPTION = "Fetches a list of chatters and updates points/time accordingly - this is required to be turned on in order for the bot to record user time and/or points"
+    NAME = "Loyalty"
+    DESCRIPTION = "Earn currency and build your watchtime while you watch your favourite streamer!"
```
Comment (pajlada) [inline pajbot/modules/chatters_refresh.py]: Call it points until the points/currency overhaul

**Comment by alazymeme** [comment]
Those commits
image

**Pull request merged**